<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendeEffektWindow</class>
 <widget class="QDialog" name="SendeEffektWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushSenden">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Auswahl Senden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushAnhalten">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-5</number>
   </property>
   <property name="text">
    <string>Alle Anhalten</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="IPAdresseLabel">
      <property name="text">
       <string>IP-Adresse</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="IPAdresseLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
